DPGAnalysis/MuonSysAging/plugins/ME0ChamberMasker.cc

```diff
@@ -0,0 +1,182 @@
+// -*- C++ -*-
+// Class:      ME0ChamberMasker
+// 
+
+
+// system include files
+#include <memory>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <algorithm>
+#include <regex>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "Geometry/GEMGeometry/interface/ME0Geometry.h"
+#include "Geometry/GEMGeometry/interface/ME0EtaPartitionSpecs.h"
+#include "Geometry/CommonDetUnit/interface/GeomDet.h"
+#include "DataFormats/GEMDigi/interface/ME0DigiPreRecoCollection.h"
+#include "CondFormats/MuonSystemAging/interface/MuonSystemAging.h"
+#include "CondFormats/DataRecord/interface/MuonSystemAgingRcd.h"
+//
+// class declaration
+//
+
+class ME0ChamberMasker : public edm::EDProducer 
```

Comment:
This class needs to be a modern producer (probably an `edm::stream::EDProducer`)